crypto/engine/eng_rdrand.c

```diff
@@ -20,22 +20,21 @@
      defined(__x86_64) || defined(__x86_64__) || \
      defined(_M_AMD64) || defined (_M_X64)) && defined(OPENSSL_CPUID_OBJ)
 
-size_t OPENSSL_ia32_rdrand(void);
+int OPENSSL_ia32_rdrand(size_t *rnd);
```

Comment:
Do *NOT* change OPENSSL_ia32_rdrand, just switch to OPENSSL_ia32_rdrand_bytes.

Reply:
Just saw this comment earlier on in the thread. The main reason I chose to do this is in case someone wanted to backport the change to prior versions. If that's not going to happen I can flip it over to _bytes instead.

Reply:
For starters which earlier version? Secondly, if it has to be the case, one would be better off porting _bytes.